travis-scripts/script.sh

```diff
@@ -36,7 +40,7 @@ then
     make CFLAGS="-Werror -Wno-pointer-sign" -k
 elif [ "$JOB_TYPE" = compile_and_unit_test ]
 then
-    make CFLAGS="-Werror -Wno-pointer-sign"  &&
+    make CFLAGS="-Werror -Wno-pointer-sign"
```

Comment:
Why this change?

Reply:
It seemed unneccessary, because of `set -e`?